extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -692,9 +695,38 @@ export const HistoryItem = ({
           operationText: operationString,
         }));
       }
+
+      if (transactionSuccessful === false) {
```

Comment:
at the very end of the decision tree, simply overwrite the icon and the tx details header. This preserves all the of the manipulation we did for (for ex) setting payment information in tx details